src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.commands;
+
+
+/**
+ * Sorts all people in the address book by name in ascneding/descending lexicographical order.
```

Comment:
Small typo here (ascending)

Reply:
Opps thanks for pointing that out .

> Will their arrangement change/remain when sorted from lexicographical to co-lexicographical, vice versa?

Persons of the same name will remain in the order they were entered into the address book as ensured by the sorting algorithm. I agree that this information should indeed be documented.